certified-connectors/TOPdesk/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Login Name",
+          "description": "The Login Name for this api",
+          "tooltip": "Provide the Login Name",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Application Password",
+          "description": "The Application Password for this api",
+          "tooltip": "Provide the Application Password",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```

Comment:
@CJusjong Did you read the checklist mentioned in the PR description ?

![image](https://user-images.githubusercontent.com/56276394/125002331-8a9ffb80-e009-11eb-8f9f-685cb09608ce.png)